docs/keyboardShortcuts.md

```diff
@@ -62,8 +62,9 @@ These shortcuts can be used when the focus is on an issue card in a panel.
 
 ## Control the panel view window ...
 
-| **Action on the browser view**| **Shortcut**|**Explanation**|
+| **Action on the panel view**| **Shortcut**|**Explanation**|
 | ------------- |:-------------:| -----------|
+| Bring HubTurbo into focus|<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>H</kbd>| Global Hotkey to make HubTurbo topmost window|
```

Comment:
It wont work at all on other platforms? Or work partially? At least the restore part (i.e. the current behavior of Ctrl+Alt+H is supposed to work on all platforms right?)